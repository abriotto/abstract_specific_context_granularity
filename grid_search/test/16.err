  0%|                                                                                                                                                                                                    | 0/99 [00:00<?, ?it/s] 10%|##################8                                                                                                                                                                        | 10/99 [00:00<00:01, 79.48it/s] 20%|#####################################7                                                                                                                                                     | 20/99 [00:00<00:00, 85.89it/s] 29%|######################################################7                                                                                                                                    | 29/99 [00:00<00:00, 79.07it/s] 37%|#####################################################################8                                                                                                                     | 37/99 [00:00<00:00, 69.10it/s] 45%|#####################################################################################                                                                                                      | 45/99 [00:00<00:00, 71.56it/s] 54%|####################################################################################################1                                                                                      | 53/99 [00:00<00:00, 65.35it/s] 61%|#################################################################################################################3                                                                         | 60/99 [00:00<00:00, 66.26it/s] 68%|##############################################################################################################################5                                                            | 67/99 [00:00<00:00, 65.50it/s] 77%|###############################################################################################################################################5                                           | 76/99 [00:01<00:00, 70.42it/s] 85%|##############################################################################################################################################################6                            | 84/99 [00:01<00:00, 62.93it/s] 92%|###########################################################################################################################################################################8               | 91/99 [00:01<00:00, 63.12it/s] 99%|#########################################################################################################################################################################################1 | 98/99 [00:01<00:00, 63.27it/s]100%|###########################################################################################################################################################################################| 99/99 [00:01<00:00, 68.19it/s]
  0%|                                                                                                                                                                                                    | 0/25 [00:00<?, ?it/s] 28%|####################################################6                                                                                                                                       | 7/25 [00:00<00:00, 63.71it/s] 60%|################################################################################################################2                                                                          | 15/25 [00:00<00:00, 63.59it/s] 88%|####################################################################################################################################################################5                      | 22/25 [00:00<00:00, 64.56it/s]100%|###########################################################################################################################################################################################| 25/25 [00:00<00:00, 61.16it/s]
