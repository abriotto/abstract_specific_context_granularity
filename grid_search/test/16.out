# ["--attributes=3", "--values=4", "--game_size=3", "--vocab_size_factor=3", "--n_epochs=100", "--batch_size=32", "--learning_rate=0.001", "--hidden_size=128", "--temp_update=0.99", "--temperature=1.5"]
Creating train_ds and val_ds...

Creating test_ds...
{"loss": 0.6962503790855408, "acc": 0.5210378766059875, "length": 3.553593873977661, "mode": "train", "epoch": 1}
{"loss": 0.6954494714736938, "acc": 0.49720674753189087, "length": 1.0, "mode": "test", "epoch": 1}
{"loss": 0.6880539059638977, "acc": 0.5557671189308167, "length": 3.3934595584869385, "mode": "train", "epoch": 2}
{"loss": 0.6922434568405151, "acc": 0.5085544586181641, "length": 1.083798885345459, "mode": "test", "epoch": 2}
{"loss": 0.6796733140945435, "acc": 0.5819500088691711, "length": 3.153770685195923, "mode": "train", "epoch": 3}
{"loss": 0.6644091606140137, "acc": 0.6137640476226807, "length": 3.614525079727173, "mode": "test", "epoch": 3}
{"loss": 0.6302746534347534, "acc": 0.6634775996208191, "length": 3.707056760787964, "mode": "train", "epoch": 4}
{"loss": 0.6213211417198181, "acc": 0.6778398752212524, "length": 4.0, "mode": "test", "epoch": 4}
{"loss": 0.5780469179153442, "acc": 0.7009711265563965, "length": 3.9091970920562744, "mode": "train", "epoch": 5}
{"loss": 0.6305396556854248, "acc": 0.6750465035438538, "length": 4.0, "mode": "test", "epoch": 5}
{"loss": 0.5477067232131958, "acc": 0.7389554977416992, "length": 3.944150686264038, "mode": "train", "epoch": 6}
{"loss": 0.5926754474639893, "acc": 0.7160150408744812, "length": 4.0, "mode": "test", "epoch": 6}
{"loss": 0.510573148727417, "acc": 0.759584903717041, "length": 3.9687998294830322, "mode": "train", "epoch": 7}
{"loss": 0.583806037902832, "acc": 0.7299814820289612, "length": 4.0, "mode": "test", "epoch": 7}
{"loss": 0.4861268997192383, "acc": 0.7719730734825134, "length": 3.972996234893799, "mode": "train", "epoch": 8}
{"loss": 0.5894466638565063, "acc": 0.725325882434845, "length": 4.0, "mode": "test", "epoch": 8}
{"loss": 0.47169461846351624, "acc": 0.7800099849700928, "length": 3.9840478897094727, "mode": "train", "epoch": 9}
{"loss": 0.5933505892753601, "acc": 0.7374301552772522, "length": 4.0, "mode": "test", "epoch": 9}
{"loss": 0.4605456292629242, "acc": 0.7726696133613586, "length": 3.9852027893066406, "mode": "train", "epoch": 10}
{"loss": 0.5846222043037415, "acc": 0.7448789477348328, "length": 4.0, "mode": "test", "epoch": 10}
{"loss": 0.4513600468635559, "acc": 0.7918379306793213, "length": 3.9875662326812744, "mode": "train", "epoch": 11}
{"loss": 0.6008121371269226, "acc": 0.7327746748924255, "length": 4.0, "mode": "test", "epoch": 11}
{"loss": 0.4479238986968994, "acc": 0.8005252480506897, "length": 3.9903311729431152, "mode": "train", "epoch": 12}
{"loss": 0.5817034244537354, "acc": 0.7318435907363892, "length": 4.0, "mode": "test", "epoch": 12}
{"loss": 0.43987300992012024, "acc": 0.7962414026260376, "length": 3.9907591342926025, "mode": "train", "epoch": 13}
{"loss": 0.5892069935798645, "acc": 0.7383611798286438, "length": 4.0, "mode": "test", "epoch": 13}
{"loss": 0.4265303909778595, "acc": 0.8127244710922241, "length": 3.9926209449768066, "mode": "train", "epoch": 14}
{"loss": 0.5761154890060425, "acc": 0.753259003162384, "length": 4.0, "mode": "test", "epoch": 14}
{"loss": 0.4250839650630951, "acc": 0.818625807762146, "length": 3.992462635040283, "mode": "train", "epoch": 15}
{"loss": 0.5707395672798157, "acc": 0.7625698447227478, "length": 4.0, "mode": "test", "epoch": 15}
{"loss": 0.4060361981391907, "acc": 0.8235921859741211, "length": 3.99029278755188, "mode": "train", "epoch": 16}
{"loss": 0.5742872953414917, "acc": 0.75698322057724, "length": 4.0, "mode": "test", "epoch": 16}
{"loss": 0.39609721302986145, "acc": 0.8338512778282166, "length": 3.992213010787964, "mode": "train", "epoch": 17}
{"loss": 0.5572624802589417, "acc": 0.7560521364212036, "length": 4.0, "mode": "test", "epoch": 17}
{"loss": 0.38247573375701904, "acc": 0.8438108563423157, "length": 3.991525411605835, "mode": "train", "epoch": 18}
{"loss": 0.5744531154632568, "acc": 0.7439478635787964, "length": 4.0, "mode": "test", "epoch": 18}
{"loss": 0.3694612383842468, "acc": 0.8491045832633972, "length": 3.992202043533325, "mode": "train", "epoch": 19}
{"loss": 0.582017719745636, "acc": 0.7392924427986145, "length": 4.0, "mode": "test", "epoch": 19}
{"loss": 0.3592427372932434, "acc": 0.8553286790847778, "length": 3.9935295581817627, "mode": "train", "epoch": 20}
{"loss": 0.5432868003845215, "acc": 0.7746742367744446, "length": 4.0, "mode": "test", "epoch": 20}
{"loss": 0.3513113260269165, "acc": 0.8533780574798584, "length": 3.9902143478393555, "mode": "train", "epoch": 21}
{"loss": 0.5470618605613708, "acc": 0.7700185775756836, "length": 4.0, "mode": "test", "epoch": 21}
{"loss": 0.34807923436164856, "acc": 0.8584473133087158, "length": 3.9912633895874023, "mode": "train", "epoch": 22}
{"loss": 0.5298111438751221, "acc": 0.769087553024292, "length": 4.0, "mode": "test", "epoch": 22}
{"loss": 0.3345879912376404, "acc": 0.8653263449668884, "length": 3.9949100017547607, "mode": "train", "epoch": 23}
{"loss": 0.5238218307495117, "acc": 0.7895716428756714, "length": 4.0, "mode": "test", "epoch": 23}
{"loss": 0.31375187635421753, "acc": 0.8728111982345581, "length": 3.9957587718963623, "mode": "train", "epoch": 24}
{"loss": 0.5000852346420288, "acc": 0.7858474254608154, "length": 4.0, "mode": "test", "epoch": 24}
{"loss": 0.3101859986782074, "acc": 0.8743727803230286, "length": 3.9967567920684814, "mode": "train", "epoch": 25}
{"loss": 0.5186519622802734, "acc": 0.787709653377533, "length": 4.0, "mode": "test", "epoch": 25}
{"loss": 0.30239954590797424, "acc": 0.8809170126914978, "length": 3.9966888427734375, "mode": "train", "epoch": 26}
{"loss": 0.4906916320323944, "acc": 0.7886406183242798, "length": 4.0, "mode": "test", "epoch": 26}
{"loss": 0.29215505719184875, "acc": 0.8818740248680115, "length": 3.99712872505188, "mode": "train", "epoch": 27}
{"loss": 0.5082356333732605, "acc": 0.7858473062515259, "length": 4.0, "mode": "test", "epoch": 27}
{"loss": 0.28711187839508057, "acc": 0.8884068131446838, "length": 3.9971213340759277, "mode": "train", "epoch": 28}
{"loss": 0.4862402081489563, "acc": 0.795158326625824, "length": 4.0, "mode": "test", "epoch": 28}
{"loss": 0.2847248613834381, "acc": 0.8835698962211609, "length": 3.993112087249756, "mode": "train", "epoch": 29}
{"loss": 0.4545897841453552, "acc": 0.8137803673744202, "length": 4.0, "mode": "test", "epoch": 29}
{"loss": 0.27131640911102295, "acc": 0.8911195993423462, "length": 3.993776559829712, "mode": "train", "epoch": 30}
{"loss": 0.48796841502189636, "acc": 0.8035382032394409, "length": 4.0, "mode": "test", "epoch": 30}
{"loss": 0.26288583874702454, "acc": 0.8962267637252808, "length": 3.9982776641845703, "mode": "train", "epoch": 31}
{"loss": 0.4963141083717346, "acc": 0.7960895299911499, "length": 4.0, "mode": "test", "epoch": 31}
{"loss": 0.2575591206550598, "acc": 0.8971661329269409, "length": 3.998765230178833, "mode": "train", "epoch": 32}
{"loss": 0.43285858631134033, "acc": 0.819366991519928, "length": 4.0, "mode": "test", "epoch": 32}
{"loss": 0.25579187273979187, "acc": 0.9018364548683167, "length": 3.9988510608673096, "mode": "train", "epoch": 33}
{"loss": 0.46948298811912537, "acc": 0.8091248273849487, "length": 4.0, "mode": "test", "epoch": 33}
{"loss": 0.24084968864917755, "acc": 0.9064528942108154, "length": 3.9965126514434814, "mode": "train", "epoch": 34}
{"loss": 0.47048985958099365, "acc": 0.8258845210075378, "length": 4.0, "mode": "test", "epoch": 34}
{"loss": 0.2384866327047348, "acc": 0.9080550074577332, "length": 3.9986414909362793, "mode": "train", "epoch": 35}
{"loss": 0.4417386054992676, "acc": 0.8296088576316833, "length": 4.0, "mode": "test", "epoch": 35}
{"loss": 0.23299263417720795, "acc": 0.9133548140525818, "length": 3.9987289905548096, "mode": "train", "epoch": 36}
{"loss": 0.4402947425842285, "acc": 0.8286778926849365, "length": 4.0, "mode": "test", "epoch": 36}
{"loss": 0.22657126188278198, "acc": 0.9151967763900757, "length": 3.9979519844055176, "mode": "train", "epoch": 37}
{"loss": 0.46107107400894165, "acc": 0.826815664768219, "length": 4.0, "mode": "test", "epoch": 37}
{"loss": 0.21652862429618835, "acc": 0.9189677834510803, "length": 3.998676061630249, "mode": "train", "epoch": 38}
{"loss": 0.46530914306640625, "acc": 0.8333333730697632, "length": 4.0, "mode": "test", "epoch": 38}
{"loss": 0.2181347906589508, "acc": 0.9105430841445923, "length": 3.998234987258911, "mode": "train", "epoch": 39}
{"loss": 0.4660029113292694, "acc": 0.8240225315093994, "length": 4.0, "mode": "test", "epoch": 39}
{"loss": 0.20520637929439545, "acc": 0.9227110743522644, "length": 3.999114751815796, "mode": "train", "epoch": 40}
{"loss": 0.41492530703544617, "acc": 0.8333333730697632, "length": 4.0, "mode": "test", "epoch": 40}
{"loss": 0.20310284197330475, "acc": 0.9192748069763184, "length": 3.998796224594116, "mode": "train", "epoch": 41}
{"loss": 0.45019978284835815, "acc": 0.833333432674408, "length": 4.0, "mode": "test", "epoch": 41}
{"loss": 0.2057148814201355, "acc": 0.9229640960693359, "length": 3.997041940689087, "mode": "train", "epoch": 42}
{"loss": 0.4824012517929077, "acc": 0.8277467489242554, "length": 4.0, "mode": "test", "epoch": 42}
{"loss": 0.2009757161140442, "acc": 0.9199162125587463, "length": 3.9991321563720703, "mode": "train", "epoch": 43}
{"loss": 0.43627631664276123, "acc": 0.8500930666923523, "length": 4.0, "mode": "test", "epoch": 43}
{"loss": 0.19391989707946777, "acc": 0.928318440914154, "length": 3.999077320098877, "mode": "train", "epoch": 44}
{"loss": 0.44301533699035645, "acc": 0.8454376459121704, "length": 4.0, "mode": "test", "epoch": 44}
{"loss": 0.1924384981393814, "acc": 0.9304892420768738, "length": 3.998981475830078, "mode": "train", "epoch": 45}
{"loss": 0.45476019382476807, "acc": 0.8379889130592346, "length": 4.0, "mode": "test", "epoch": 45}
{"loss": 0.19326716661453247, "acc": 0.9273967146873474, "length": 3.999513626098633, "mode": "train", "epoch": 46}
{"loss": 0.4319990277290344, "acc": 0.8510242700576782, "length": 4.0, "mode": "test", "epoch": 46}
{"loss": 0.18642719089984894, "acc": 0.932375431060791, "length": 3.9993948936462402, "mode": "train", "epoch": 47}
{"loss": 0.4386337399482727, "acc": 0.8482309579849243, "length": 4.0, "mode": "test", "epoch": 47}
{"loss": 0.18645548820495605, "acc": 0.9313750267028809, "length": 3.996746301651001, "mode": "train", "epoch": 48}
{"loss": 0.5071078538894653, "acc": 0.8258845806121826, "length": 4.0, "mode": "test", "epoch": 48}
{"loss": 0.1775851547718048, "acc": 0.9351816773414612, "length": 3.999504327774048, "mode": "train", "epoch": 49}
{"loss": 0.4267200231552124, "acc": 0.8510242700576782, "length": 4.0, "mode": "test", "epoch": 49}
{"loss": 0.1829916089773178, "acc": 0.9382902383804321, "length": 3.9993624687194824, "mode": "train", "epoch": 50}
{"loss": 0.40581583976745605, "acc": 0.8547486662864685, "length": 4.0, "mode": "test", "epoch": 50}
{"loss": 0.17281152307987213, "acc": 0.9389185309410095, "length": 3.99942946434021, "mode": "train", "epoch": 51}
{"loss": 0.4261525571346283, "acc": 0.8556798100471497, "length": 4.0, "mode": "test", "epoch": 51}
{"loss": 0.17612770199775696, "acc": 0.9376769065856934, "length": 3.9992096424102783, "mode": "train", "epoch": 52}
{"loss": 0.4199877977371216, "acc": 0.8510242700576782, "length": 4.0, "mode": "test", "epoch": 52}
{"loss": 0.157803013920784, "acc": 0.9447975754737854, "length": 3.997469663619995, "mode": "train", "epoch": 53}
{"loss": 0.44447970390319824, "acc": 0.8463687896728516, "length": 4.0, "mode": "test", "epoch": 53}
{"loss": 0.1640797108411789, "acc": 0.9383065700531006, "length": 3.9996883869171143, "mode": "train", "epoch": 54}
{"loss": 0.44978201389312744, "acc": 0.8510242700576782, "length": 4.0, "mode": "test", "epoch": 54}
{"loss": 0.1550338864326477, "acc": 0.9417330026626587, "length": 3.9996416568756104, "mode": "train", "epoch": 55}
{"loss": 0.4150806665420532, "acc": 0.8538176417350769, "length": 4.0, "mode": "test", "epoch": 55}
{"loss": 0.14899787306785583, "acc": 0.9482749700546265, "length": 3.999606132507324, "mode": "train", "epoch": 56}
{"loss": 0.4352034032344818, "acc": 0.8463687896728516, "length": 4.0, "mode": "test", "epoch": 56}
{"loss": 0.15354464948177338, "acc": 0.9423527121543884, "length": 3.9993648529052734, "mode": "train", "epoch": 57}
{"loss": 0.4435997009277344, "acc": 0.851024329662323, "length": 4.0, "mode": "test", "epoch": 57}
{"loss": 0.1505558341741562, "acc": 0.9404945969581604, "length": 3.999878168106079, "mode": "train", "epoch": 58}
{"loss": 0.431999146938324, "acc": 0.8463687896728516, "length": 4.0, "mode": "test", "epoch": 58}
{"loss": 0.14565525949001312, "acc": 0.9448108077049255, "length": 3.9991018772125244, "mode": "train", "epoch": 59}
{"loss": 0.4666019082069397, "acc": 0.8500933051109314, "length": 4.0, "mode": "test", "epoch": 59}
{"loss": 0.15365205705165863, "acc": 0.9398490786552429, "length": 3.9991543292999268, "mode": "train", "epoch": 60}
{"loss": 0.45708101987838745, "acc": 0.8426443934440613, "length": 4.0, "mode": "test", "epoch": 60}
{"loss": 0.15643210709095, "acc": 0.9392468333244324, "length": 3.9998252391815186, "mode": "train", "epoch": 61}
{"loss": 0.43620380759239197, "acc": 0.8538175821304321, "length": 4.0, "mode": "test", "epoch": 61}
{"loss": 0.14930471777915955, "acc": 0.9389347434043884, "length": 3.9998221397399902, "mode": "train", "epoch": 62}
{"loss": 0.4649423062801361, "acc": 0.8603351712226868, "length": 4.0, "mode": "test", "epoch": 62}
{"loss": 0.13412785530090332, "acc": 0.9519833326339722, "length": 3.9992332458496094, "mode": "train", "epoch": 63}
{"loss": 0.43568119406700134, "acc": 0.8575417995452881, "length": 4.0, "mode": "test", "epoch": 63}
{"loss": 0.13751576840877533, "acc": 0.9507694840431213, "length": 3.9997901916503906, "mode": "train", "epoch": 64}
{"loss": 0.4531937539577484, "acc": 0.8640596866607666, "length": 4.0, "mode": "test", "epoch": 64}
{"loss": 0.14374174177646637, "acc": 0.9445437788963318, "length": 3.9998576641082764, "mode": "train", "epoch": 65}
{"loss": 0.43754929304122925, "acc": 0.8696463108062744, "length": 4.0, "mode": "test", "epoch": 65}
{"loss": 0.1340152621269226, "acc": 0.9498369097709656, "length": 3.9998586177825928, "mode": "train", "epoch": 66}
{"loss": 0.45743662118911743, "acc": 0.8547486066818237, "length": 4.0, "mode": "test", "epoch": 66}
{"loss": 0.1298137605190277, "acc": 0.9525913000106812, "length": 3.9987218379974365, "mode": "train", "epoch": 67}
{"loss": 0.46692341566085815, "acc": 0.8584731817245483, "length": 4.0, "mode": "test", "epoch": 67}
{"loss": 0.13497832417488098, "acc": 0.9504603147506714, "length": 3.999412775039673, "mode": "train", "epoch": 68}
{"loss": 0.4535505175590515, "acc": 0.8491621017456055, "length": 4.0, "mode": "test", "epoch": 68}
{"loss": 0.12447214126586914, "acc": 0.9545146226882935, "length": 3.99993896484375, "mode": "train", "epoch": 69}
{"loss": 0.4865188002586365, "acc": 0.8463687896728516, "length": 4.0, "mode": "test", "epoch": 69}
{"loss": 0.13497304916381836, "acc": 0.9504600167274475, "length": 3.9998722076416016, "mode": "train", "epoch": 70}
{"loss": 0.4632614552974701, "acc": 0.8407822251319885, "length": 4.0, "mode": "test", "epoch": 70}
{"loss": 0.14234235882759094, "acc": 0.9482654333114624, "length": 3.99884033203125, "mode": "train", "epoch": 71}
{"loss": 0.46232661604881287, "acc": 0.8482308983802795, "length": 4.0, "mode": "test", "epoch": 71}
{"loss": 0.12782828509807587, "acc": 0.9548264145851135, "length": 3.999927520751953, "mode": "train", "epoch": 72}
{"loss": 0.4363192915916443, "acc": 0.8612663149833679, "length": 4.0, "mode": "test", "epoch": 72}
{"loss": 0.11810782551765442, "acc": 0.9616759419441223, "length": 3.9998586177825928, "mode": "train", "epoch": 73}
{"loss": 0.46116161346435547, "acc": 0.8500933051109314, "length": 4.0, "mode": "test", "epoch": 73}
{"loss": 0.12747332453727722, "acc": 0.9569929242134094, "length": 3.9995992183685303, "mode": "train", "epoch": 74}
{"loss": 0.475763201713562, "acc": 0.8407822251319885, "length": 4.0, "mode": "test", "epoch": 74}
{"loss": 0.12346760183572769, "acc": 0.9578263163566589, "length": 3.997913122177124, "mode": "train", "epoch": 75}
{"loss": 0.4391133487224579, "acc": 0.8575419187545776, "length": 4.0, "mode": "test", "epoch": 75}
{"loss": 0.11738910526037216, "acc": 0.9591864347457886, "length": 3.9999165534973145, "mode": "train", "epoch": 76}
{"loss": 0.43115586042404175, "acc": 0.864990770816803, "length": 4.0, "mode": "test", "epoch": 76}
{"loss": 0.12982316315174103, "acc": 0.9526420831680298, "length": 3.9998812675476074, "mode": "train", "epoch": 77}
{"loss": 0.45393216609954834, "acc": 0.8575419783592224, "length": 4.0, "mode": "test", "epoch": 77}
{"loss": 0.11777548491954803, "acc": 0.9576317667961121, "length": 3.999974489212036, "mode": "train", "epoch": 78}
{"loss": 0.4633218050003052, "acc": 0.8510242700576782, "length": 4.0, "mode": "test", "epoch": 78}
{"loss": 0.10434474796056747, "acc": 0.9666649699211121, "length": 3.999910593032837, "mode": "train", "epoch": 79}
{"loss": 0.4754582345485687, "acc": 0.8454375863075256, "length": 4.0, "mode": "test", "epoch": 79}
{"loss": 0.10154668986797333, "acc": 0.9672815203666687, "length": 3.999807834625244, "mode": "train", "epoch": 80}
{"loss": 0.4501464366912842, "acc": 0.8491621017456055, "length": 4.0, "mode": "test", "epoch": 80}
{"loss": 0.1043102964758873, "acc": 0.965415894985199, "length": 3.99989652633667, "mode": "train", "epoch": 81}
{"loss": 0.4616149663925171, "acc": 0.8612664341926575, "length": 4.0, "mode": "test", "epoch": 81}
{"loss": 0.10299523919820786, "acc": 0.9644842743873596, "length": 3.9999725818634033, "mode": "train", "epoch": 82}
{"loss": 0.485348105430603, "acc": 0.8519554138183594, "length": 4.0, "mode": "test", "epoch": 82}
{"loss": 0.10589606314897537, "acc": 0.9679113626480103, "length": 3.9999794960021973, "mode": "train", "epoch": 83}
{"loss": 0.42190515995025635, "acc": 0.8594039678573608, "length": 4.0, "mode": "test", "epoch": 83}
{"loss": 0.10554098337888718, "acc": 0.9632372260093689, "length": 3.9999430179595947, "mode": "train", "epoch": 84}
{"loss": 0.48221510648727417, "acc": 0.8556798100471497, "length": 4.0, "mode": "test", "epoch": 84}
{"loss": 0.0964910015463829, "acc": 0.9660327434539795, "length": 3.9998183250427246, "mode": "train", "epoch": 85}
{"loss": 0.46824121475219727, "acc": 0.8584731817245483, "length": 4.0, "mode": "test", "epoch": 85}
{"loss": 0.10390985757112503, "acc": 0.9626163840293884, "length": 3.9999876022338867, "mode": "train", "epoch": 86}
{"loss": 0.47881484031677246, "acc": 0.8538176417350769, "length": 4.0, "mode": "test", "epoch": 86}
{"loss": 0.10015204548835754, "acc": 0.9688456058502197, "length": 3.9999589920043945, "mode": "train", "epoch": 87}
{"loss": 0.47850149869918823, "acc": 0.8510241508483887, "length": 4.0, "mode": "test", "epoch": 87}
{"loss": 0.10624685138463974, "acc": 0.9657312035560608, "length": 3.999981641769409, "mode": "train", "epoch": 88}
{"loss": 0.4819716215133667, "acc": 0.8519552946090698, "length": 4.0, "mode": "test", "epoch": 88}
{"loss": 0.12153679877519608, "acc": 0.9570077657699585, "length": 3.9997153282165527, "mode": "train", "epoch": 89}
{"loss": 0.43515968322753906, "acc": 0.8566108345985413, "length": 4.0, "mode": "test", "epoch": 89}
{"loss": 0.11455152928829193, "acc": 0.9601223468780518, "length": 3.99994158744812, "mode": "train", "epoch": 90}
{"loss": 0.5028260946273804, "acc": 0.8482309579849243, "length": 4.0, "mode": "test", "epoch": 90}
{"loss": 0.11113177984952927, "acc": 0.9629243016242981, "length": 3.9998722076416016, "mode": "train", "epoch": 91}
{"loss": 0.46106356382369995, "acc": 0.8649908304214478, "length": 4.0, "mode": "test", "epoch": 91}
{"loss": 0.10206706821918488, "acc": 0.9641738533973694, "length": 3.999988555908203, "mode": "train", "epoch": 92}
{"loss": 0.5130426287651062, "acc": 0.8575419783592224, "length": 4.0, "mode": "test", "epoch": 92}
{"loss": 0.10977859050035477, "acc": 0.9601240754127502, "length": 3.9999914169311523, "mode": "train", "epoch": 93}
{"loss": 0.4269176125526428, "acc": 0.8677840232849121, "length": 4.0, "mode": "test", "epoch": 93}
{"loss": 0.11948774755001068, "acc": 0.9572880268096924, "length": 3.9993884563446045, "mode": "train", "epoch": 94}
{"loss": 0.46427834033966064, "acc": 0.8696463108062744, "length": 4.0, "mode": "test", "epoch": 94}
{"loss": 0.09938644617795944, "acc": 0.9663545489311218, "length": 3.9999849796295166, "mode": "train", "epoch": 95}
{"loss": 0.46956512331962585, "acc": 0.872439444065094, "length": 4.0, "mode": "test", "epoch": 95}
{"loss": 0.1028658077120781, "acc": 0.9601239562034607, "length": 3.9999759197235107, "mode": "train", "epoch": 96}
{"loss": 0.4577674865722656, "acc": 0.8659218549728394, "length": 4.0, "mode": "test", "epoch": 96}
{"loss": 0.10918950289487839, "acc": 0.9619936943054199, "length": 3.99999737739563, "mode": "train", "epoch": 97}
{"loss": 0.4572542607784271, "acc": 0.8668529391288757, "length": 4.0, "mode": "test", "epoch": 97}
{"loss": 0.0997888520359993, "acc": 0.9638335108757019, "length": 3.999251127243042, "mode": "train", "epoch": 98}
{"loss": 0.45092424750328064, "acc": 0.8715084791183472, "length": 4.0, "mode": "test", "epoch": 98}
{"loss": 0.08884802460670471, "acc": 0.9694695472717285, "length": 3.999985456466675, "mode": "train", "epoch": 99}
{"loss": 0.45508915185928345, "acc": 0.8659218549728394, "length": 4.0, "mode": "test", "epoch": 99}
{"loss": 0.09756483882665634, "acc": 0.9707165956497192, "length": 3.9999990463256836, "mode": "train", "epoch": 100}
{"loss": 0.4639262855052948, "acc": 0.8696463108062744, "length": 4.0, "mode": "test", "epoch": 100}
test accuracy: 0.7948029041290283
